src/main/java/uk/gov/ons/census/casesvc/service/FulfilmentRequestService.java

```diff
@@ -56,15 +65,18 @@ public void processFulfilmentRequest(ResponseManagementEvent fulfilmentRequest)
         convertObjectToJson(fulfilmentRequestPayload));
 
     if (individualResponseRequestCodes.contains(fulfilmentRequestPayload.getFulfilmentCode())) {
-      Case individualResponseCase = prepareIndividualResponseCaseFromParentCase(caze);
+      Case individualResponseCase = prepareIndividualResponseCase(caze, fulfilmentRequest);
       caseService.saveAndEmitCaseCreatedEvent(individualResponseCase);
     }
   }
 
-  private Case prepareIndividualResponseCaseFromParentCase(Case parentCase) {
+  private Case prepareIndividualResponseCase(
+      Case parentCase, ResponseManagementEvent fulfilmentRequest) {
```

Comment:
do you really need to pass the whole ResponseManagementEvent into this or could you just pass the individual case ID?

Reply:
changed it without quibblizing, partly because it will need just Id when merged in with Ade's changes.  This prepare function has been moved to caseService as re-used.